src/modules/m_sqloper.cpp

```diff
@@ -99,7 +99,7 @@ class OperQuery : public SQL::Query
 			OperInfo* ifo = new OperInfo(type);
 
 			ifo->type_block = tblk->second->type_block;
-			ifo->oper_block = tag;
+			ifo->oper_block = std::move(tag);
```

Comment:
The v3 branch has to build as GNU++03 so `std::move` can't be used. This can be committed to master though.

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4